javatests/arcs/core/allocator/DefaultStorageKeyCreatorTest.kt

```diff
@@ -0,0 +1,83 @@
+package arcs.core.allocator
+
+import arcs.core.common.Id
+import arcs.core.data.CreatableStorageKey
+import arcs.core.data.Plan
+import arcs.core.host.MultiplePersonPlan
+import arcs.core.host.PersonPlan
+import arcs.core.storage.api.DriverAndKeyConfigurator
+import arcs.core.storage.driver.RamDisk
+import com.google.common.truth.Truth
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+@OptIn(ExperimentalCoroutinesApi::class)
+class DefaultStorageKeyCreatorTest {
+
+  @Before
+  open fun setUp() = runBlocking {
+    RamDisk.clear()
+    DriverAndKeyConfigurator.configure(null)
+  }
+
+  // Simple Case.
+  @Test
+  fun keyCreator_createStorageKeysIfNecessary() = creatorKeyTest(PersonPlan)
```

Comment:
maybe add the plans as comments to the top of each test, so its easy to see what's going on?

Reply:
The recipes are rather long (especially for MultiplePersonPlan) so I noted where to find the Arcs Manifest File.

Reply:
I don't think 24 lines (including empty lines) is too long, particularly if it helps with understanding of the test. Then again maybe that's not the case here.